core/block/editor/stext/text.go

```diff
@@ -462,6 +472,29 @@ func (t *textImpl) TurnInto(ctx session.Context, style model.BlockContentTextSty
 	return t.Apply(s)
 }
 
+func (t *textImpl) isLastTextBlockChanged() (bool, error) {
+	if t.lastSetTextState == nil || t.lastSetTextId == "" {
+		return true, fmt.Errorf("last state about text block is not saved")
+	}
+	newTextBlock, err := getText(t.lastSetTextState, t.lastSetTextId)
+	if err != nil {
+		return true, err
+	}
+	oldTextBlock := t.Pick(t.lastSetTextId)
```

Comment:
seems like you could use `t.lastSetTextState.PickOrigin` (or get parent state and pick block from it without using helper PickOrigin), looks little bit confusing that we compare blocks from what is looks like different states. Sure they have parent-child relationship (t is parent of lastSetTextState), but more transparency won't hurt

Reply:
Yeah, used PickOrigin for retrieving old state - need to fix tests though